pipeline/data_merging/getMupitStructure.py

```diff
@@ -0,0 +1,134 @@
+import requests
+import json
+import sys
+import argparse
+import csv
+import time
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Determine correct BRCA structure from MuPIT.')
+    parser.add_argument('-i', '--input', type=argparse.FileType('r'),
+                        help='Input variants.')
+    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+                        help='Output variants.')
+    options = parser.parse_args()
+    return options
+
+
+def isMissenseSubstitution(ref, alt):
+    bases = ['a', 'c', 't', 'g']
+    ref = ref.lower()
+    alt = alt.lower()
+    if ref in bases and alt in bases and len(ref) == 1 and len(alt) == 1:
+        return True
+    return False
+
+
+def main(args):
+    options = parse_args()
+    inputFile = options.input
+    outputFile = options.output
+
+    input_file = csv.reader(inputFile, delimiter='\t')
+    output_file = csv.writer(outputFile, delimiter='\t')
+    input_header_row = input_file.next()
+
+    # The following new columns will contain data generated by this file
+    new_columns_to_append = ["mupit_structure"]
+
+    output_header_row = input_header_row + new_columns_to_append
+
+    output_file.writerow(output_header_row)
+
+    chromIndex = input_header_row.index("Chr")
+    posIndex = input_header_row.index("Pos")
+    refIndex = input_header_row.index("Ref")
+    altIndex = input_header_row.index("Alt")
+
+    for variant in input_file:
+        chrom = "chr" + variant[chromIndex]
+        pos = int(variant[posIndex])
+        ref = variant[refIndex]
+        alt = variant[altIndex]
+
+        # only concerned with missense substitutions
+        if isMissenseSubstitution(ref, alt):
+            # Add empty data for each new column to prepare for data insertion by index
+            for i in range(len(new_columns_to_append)):
+                variant.append('-')
+
+            retries = 5
+            if (pos >= 32356427 and pos <= 32396972) or (pos >= 43045692 and pos <= 43125184):
```

Comment:
It would be good to put these numbers into some separate module, s.t. we use consistent boundaries.